x-pack/legacy/plugins/reporting/types.d.ts

```diff
@@ -93,51 +90,6 @@ export type ReportingResponseToolkit = Legacy.ResponseToolkit;
 
 export type ESCallCluster = CallCluster;
```

Comment:
Type-wise it should be the same and there shouldn't be any issues but `CallCluster` is now `APICaller` in NP from `'src/core/server'`